packages/react-components/react-motion/stories/src/CreatePresenceComponent/CreatePresenceComponentInAndOut.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { createPresenceComponent, makeStyles, tokens } from '@fluentui/react-components';
+import * as React from 'react';
+
+import description from './CreatePresenceComponentInAndOut.stories.md';
+
+const useClasses = makeStyles({
+  container: {
+    display: 'grid',
+    gridTemplate: `"card card" "controls ." / 1fr 1fr`,
+    gap: '20px 10px',
+  },
+
+  card: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    gap: '10px',
+    gridArea: 'card',
+
+    border: `${tokens.strokeWidthThicker} solid ${tokens.colorNeutralForeground3}`,
+    borderRadius: tokens.borderRadiusMedium,
+    boxShadow: tokens.shadow16,
+    padding: '10px',
+  },
+
+  item: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: tokens.colorBrandBackground,
+    border: `${tokens.strokeWidthThicker} solid ${tokens.colorTransparentStroke}`,
+    borderRadius: '50%',
+
+    width: '100px',
+    height: '100px',
+    color: 'white',
+  },
+});
+
+const MyFade = createPresenceComponent({
+  enter: {
+    keyframes: [{ opacity: 0 }, { opacity: 1 }],
+    duration: 4000,
+    iterations: Infinity,
+  },
+
+  exit: {
+    keyframes: [{ opacity: 1 }, { opacity: 0 }],
+    duration: 2000,
+    iterations: Infinity,
+  },
```

Comment:
Plz tweak this motion for reduced motion as it can hurt eyes :)


https://github.com/user-attachments/assets/34295ef4-cee0-408d-a30d-d624bd514acd

Reply:
Good catch, done.